src/models/mail.py

```diff
@@ -1,15 +1,21 @@
-# Author: Olivier Dion - 2019
+#######################
+# Authors:            #
+#                     #
+# Olivier Dion - 2019 #
+#######################
 
 import smtplib
-
+from os.path import isfile
 from email.mime.multipart import MIMEMultipart
 from email.mime.base import MIMEBase
 from email.mime.text import MIMEText
-from os.path import isfile
+
```

Comment:
include **std**
include **lib**
include **app**

Reply:
nvm, email is from stdlib forgot

Reply:
je change ça